airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2454,6 +2454,10 @@ components:
             $ref: "#/components/schemas/Notification"
         defaultGeography:
           $ref: "#/components/schemas/Geography"
+        webhookConfigs:
```

Comment:
nice catch.

just to confirm, the other two routes we need this on are the create and update routes and it looks like we have them.

is that right?

Reply:
Yes indeed - mainly we'll only use the update route, but we want it in the create route for consistency's sake I think.

Reply:
Yeap. Consistency is good!